bin/check-rds-events.rb

```diff
@@ -20,7 +20,14 @@
 #   gem: sensu-plugin
 #
 # USAGE:
-#  ./check-rds-events.rb -r ${your_region} -k ${your_aws_secret_access_key} -a ${your_aws_access_key}
+#  Check's a specific RDS instance on a specific availability zone for critical events
+#  check-rds-events.rb -r ${your_region} -k ${your_aws_secret_access_key} -a ${your_aws_access_key} -i ${your_rds_instance_id_name}
+#
+#  Checks all RDS instances on a specific availability zone
+#  check-rds-events.rb -r ${your_region} -k ${your_aws_secret_access_key} -a ${your_aws_access_key}
```

Comment:
I don't believe you can specify the zone, you can only specify the region. This was the original behaviour and original documentation format that existed in the check. I should change the comment above though to not state `specific availability zone` and instead state `specific region`